src/components/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link';
+import { Breadcrumbs } from '@aws-amplify/ui-react';
+import { findDirectoryNode as findNode } from '@/utils/findDirectoryNode';
+
+type BreadcrumbItem = {
+  href: {pathname:string, query?: { platform: string }};
+  label: string;
+  isDisabled?: boolean;
+};
+
+type Props = {
+  route: string;
+  platform: string;
+};
+
+function generateBreadcrumbs(route:string, platform:string):BreadcrumbItem[]{
+  const breadcrumbs:BreadcrumbItem[] = [];
+
+  const pieces = route.split('/').filter((str) => str);
+  let urls:string[] = [];
+  for(let i = 1; i <= pieces.length; i++){
+    urls.push(`/${pieces.slice(0,i).join('/')}`);
+  }
+  urls.splice(0,1,"/");
+
+  urls.forEach((url) => {
+    const directoryEntry = findNode(url);
+    breadcrumbs.push({
+      href:{
+        pathname: url,
+        query: { platform: platform }
+      },
+      label: directoryEntry ? directoryEntry.title : url
```

Comment:
Should we add some logic here to catch the root path and show "Home" instead of "Amplify Docs"? I think from the designs it originally has "Home"

I think also it should only show when we're visited a path with depth 1? So it wouldn't show up on the "Home" page, but would show up on "Home > Get started" for instance